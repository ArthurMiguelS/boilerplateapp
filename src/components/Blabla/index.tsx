import * as S from './styles'

const Blabla = () => (
  <S.Wrapper>
    <h1>Blabla</h1>
  </S.Wrapper>
)

export default Blabla
