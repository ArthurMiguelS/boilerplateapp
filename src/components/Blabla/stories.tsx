import { Meta, StoryObj } from '@storybook/react'
import Blabla from '.'

export default {
  title: 'Blabla',
  component: Blabla
} as Meta

export const Default: StoryObj = {}
