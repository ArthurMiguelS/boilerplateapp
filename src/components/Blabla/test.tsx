import { render, screen } from '@testing-library/react'

import Blabla from '.'

describe('<Blabla />', () => {
  it('should render the heading', () => {
    const { container } = render(<Blabla />)

    expect(screen.getByRole('heading', { name: /Blabla/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
