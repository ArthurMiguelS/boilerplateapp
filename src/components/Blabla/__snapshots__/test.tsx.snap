// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Blabla /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Blabla
  </h1>
</main>
`;
